src/mon/MonMap.h

```diff
@@ -456,6 +456,18 @@ class MonMap {
 
   static void generate_test_instances(std::list<MonMap*>& o);
 protected:
+  /**
+   * build a monmap from a list of entity_addrvec_t's
+   *
+   * Give mons dummy names.
+   *
+   * @param addrs  list of entity_addrvec_t's
+   * @param prefix prefix to prepend to generated mon names
+   * @return 0 for success, -errno on error
+   */
+  int init_with_addrs(const std::vector<entity_addrvec_t>& addrs,
+        bool for_mkfs,
+        const std::string_view prefix);
```

Comment:
`const std::string_view` as a function parameter means about as much as `const int`. `std::string_view` is already a `const`-y view of the string/char-array.

Reply:
Addressed, all  3 instances. Thanks! :)